addons/binding/org.openhab.binding.nibeheatpump/ESH-INF/thing/f1x45.xml

```diff
@@ -54,6 +54,11 @@
 				<description>Comma separated list of registers, which are allowed to write to heat pump. E.g. 44266, 47004</description>
 				<default></default>
 			</parameter>
+            <parameter name="throttleTime" type="integer">
+                <label>Throttle incoming data</label>
```

Comment:
"Throttle Incoming Data"

We have an unwritten naming convention for labels to use a capital letter at the beginning.

Reply:
Done